cmd/testgrid-config-generator/main.go

```diff
@@ -182,6 +183,94 @@ type release struct {
 	} `json:"verify"`
 }
 
+// e2ePresubmitJobsByBranch collects statically-defined e2e presubmit jobs (e2e,
+// serial and upgrade) grouped by the targeted branch.
+func e2ePresubmitJobsByBranch(presubmitBranches []string, jobConfig *prowconfig.JobConfig) map[string][]string {
+	presubmitJobs := map[string][]string{}
+	for _, p := range jobConfig.AllStaticPresubmits(nil) {
+		// Only track mandatory jobs
+		if !p.AlwaysRun || p.Optional {
+			continue
+		}
+		// Only track presubmits for repos in the openshift org. This is intended
+		// to minimize the cost of testgrid collection and can be relaxed if
+		// necessary. openshift-priv should not be collected as it rarely sees
+		// presubmit activity.
+		if !strings.HasPrefix(p.Name, "pull-ci-openshift-") ||
```

Comment:
The ignored key in the mapping on line 190 is the org/repo, use that instead of stringified names for filtering. You can also ask for jobs for a repo.

Reply:
I get that I can ask for jobs for a repo. How do I get repos for an org?